cmake/patches/coremltools/crossplatformbuild.patch

```diff
@@ -12,17 +12,30 @@ index adc7bfcf..7b2bf9cc 100644
  #include <sys/mman.h>
  #include <sys/stat.h>
 +#endif
-
+ 
  using namespace MILBlob;
  using namespace MILBlob::Blob;
+diff --git a/mlmodel/src/MILBlob/Blob/FileWriter.hpp b/mlmodel/src/MILBlob/Blob/FileWriter.hpp
+index 2bc99403..8400fb76 100644
+--- a/mlmodel/src/MILBlob/Blob/FileWriter.hpp
++++ b/mlmodel/src/MILBlob/Blob/FileWriter.hpp
+@@ -6,7 +6,7 @@
+ #pragma once
+ 
+ #include "MILBlob/Util/Span.hpp"
+-
++#include <cstdint>
```

Comment:
nit: could we keep the convention of marking the changes with an `ORT_EDIT` comment?

Reply:
No problem

Reply:
Updated